pkgs/top-level/all-packages.nix

```
@@ -219,6 +219,8 @@ with pkgs;
 
   fission = callPackage ../development/tools/fission { };
 
+  apko = callPackage ../development/tools/apko { };
```

```suggestion
  apko = callPackage ../development/tools/apko { };
```
`apko` was added to `BUILD SUPPORT` section instead of `DEVELOPMENT / TOOLS`. Also should respect alphabetical order.

---

@06kellyjac Thoughts? Let's just get on with this PR...

---

I added apko further down than `astrolog` which I don't think is build support. There's a lot of other random stuff there. Frankly all-packages is a massive mess.
I just put apko in the first large chunk of `a` prefixed packages I found